docs/source/get_started/access_info/access_info_css.ipynb

```diff
@@ -81,7 +81,7 @@
     {
```

Comment:
<p>Maybe worth having an addtional cell using the <code>compiled_circuit()</code> method (also in the qss notebook)</p><p> </p><p>(maybe<code>target(), num_qubit(),</code>and <code>repetitions()</code> also for completeness if not shown elsewhere) </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Done with [0f33399](https://github.com/Infleqtion/client-superstaq/pull/728/commits/0f3339910cd7a6e33a9ae3ef86672e6435d5b89d)